backend/backend.go

```diff
@@ -444,7 +444,7 @@ func (backend *Backend) persistBTCAccountConfig(
 		err := backend.persistAccount(config.Account{
 			CoinCode:       coin.Code(),
 			Name:           suffixedName,
-			Code:           fmt.Sprintf("%s-%s", code, cfg.ScriptType()),
+			Code:           accounts.Code(fmt.Sprintf("%s-%s", code, cfg.ScriptType())),
```

Comment:
Please consider addting these to accounts package, something like:

```go
package accounts

// ScriptTypedCode produces an account code used in/for ...
func ScriptTypedCode(code Code, typ signing.ScriptType) Code

// FingerprintedCode makes an account code from a keystore root fingerprint.
// ... not sure where this one is used compared to ScriptTypedCode because they both
// appear in somethingPersistAccount and would be very nice to have docs explaining it here.
func FingerprintedCode(finger []byte, coinCode coinpkg.Code, acctNumber int) Code {
  return Code(fmt.Sprintf("v0-%x-%s-%d", finger, coinCode, accountNumber))
}

// ERC20Code makes an ERC20 code where ethAccountCode is token's parent account.
func ERC20Code(ethAccountCode Code, tokenCode string) Code 
```

It groups them together and adds doc comments explaining which is used where. Also, easier to trace and have rest of the code to always refer to the same calling place instead of an inlined fmt.Sprintf.

Reply:
Excellent idea. Done